linker/shared/src/main/scala/org/scalajs/linker/backend/emitter/CoreJSLib.scala

```diff
@@ -1504,7 +1513,7 @@ private[emitter] object CoreJSLib {
           Nil
         }
 
-        val clone = MethodDef(static = false, Ident(genMethodName(cloneMethodName)), Nil, None, {
+        val clone = MethodDef(static = false, genMethodIdent(cloneMethodName), Nil, None, {
```

Comment:
Actually: This also would need an original name, no? (granted, that was here before, but maybe that really points towards a `genMethodIdentForDef`.